apps/api/src/applicant/ienapplicant.util.service.ts

```diff
@@ -317,4 +386,17 @@ export class IENApplicantUtilService {
     }
     return job_title;
   }
+
+  async updateLatestStatusOnApplicant(mappedApplicantList: string[]): Promise<void> {
+    try {
+      // update applicant with latest status
+      const idsToUpdate = `'${mappedApplicantList.join("','")}'`;
```

Comment:
how about using a view for query so that we can remove status field from applicant table? it might make updating queries simpler.